iskolivery/app/controllers/users_controller.rb

```diff
@@ -36,6 +36,8 @@ def home
 
 		# get all requests not posted by and not accepted by user
 		# keep in mind null values
+		@active_requests = Assignment.where(requester_id: @user.id)
```

Comment:
This query is wrong but that's on me, not on you. This would include requests that the user has posted, but are already accepted, and those are *inactive*.  There *should* be a "status code" in Assignment that indicates whether it's available, accepted, cancelled, or already fulfilled. I'll work that out this sprint.

I also suggest moving this line below `@available_requests`, because the comment above is for that assignment.